index.bs

```diff
@@ -14,6 +14,8 @@ spec:webidl; type:dfn; text:resolve
 </pre>
 
 <pre class=anchors>
+urlPrefix: https://html.spec.whatwg.org/; spec: html
```

Comment:
Whoops, I meant to remove this before uploading the patch. Though if we land #94 first it won't matter anyways